README.md

```diff
@@ -338,10 +338,10 @@ If you fork the project on GitHub, you can run your fork's [build workflow](.git
     --plugin-dirs PATH              Path to an additional directory to search
                                     for plugins. This option can be used
                                     multiple times to add multiple directories.
-                                    Note that this currently only works for
-                                    extractor plugins; postprocessor plugins can
-                                    only be loaded from the default plugin
-                                    directories
+                                    Add "no-external" to disable searching
+                                    default external plugin directories (outside
+                                    of python environment)
+    --no-plugins                    Do not load plugins
```

Comment:
Reminder: Remove `YTDLP_NO_PLUGINS` from docs

Reply:
I disagree. Its the same as `NO_COLOR=1 yt-dlp` vs `yt-dlp --color never`. Why remove more options when they can be useful?

Reply:
`NO_COLOR` is a standard, not something we created.

But sure, since it's already implemented, I won't insist on removing docs